akka-persistence/src/main/resources/reference.conf

```diff
@@ -171,6 +171,17 @@ akka.persistence {
         reset-timeout = 60s
       }
     }
+
+  fsm {
+    # Enable periodical snapshots, where the snapshot period (number of persistent events)
+    # is specified by "snapshot-after"
+    enable-snapshot-after = off
```

Comment:
Not sure we need two properties for this.
We can use `snapshot-after = off` as default, but also support number. We have that at several other places, e.g. https://github.com/akka/akka/blob/master/akka-cluster/src/main/resources/reference.conf#L35
and that is parsed with https://github.com/akka/akka/blob/master/akka-cluster/src/main/scala/akka/cluster/ClusterSettings.scala#L62

`Option[Int]` is a good type in scala land